### ISSUE TITLE
Backend Issue

### ISSUE THREAD

**Issue by yusriltakeuchi** [meydiansyah/flutter-make-up#4]
How to use this app? I check in the file const.dart contains URL variable to local host not public

**Comment by meydiansyah** [comment]
That's right, this project is in the same package as my other repositories, please check laravel-make-up

**Comment by yusriltakeuchi** [comment]
> That's right, this project is in the same package as my other repositories, please check laravel-make-up
<img width="600" alt="Screen Shot 2021-10-01 at 17 32 36" src="https://user-images.githubusercontent.com/10339660/135606016-c681b881-37e4-485b-ba3a-b812acfc0c09.png">

I got some error from your backend when try to login.

**Comment by meydiansyah** [comment]
try to parse your url like this :
http://192.168.111.171:8000/api/login?email=user@test.com&password=12345678

**Comment by yusriltakeuchi** [comment]
> try to parse your url like this :
> http://192.168.111.171:8000/api/login?email=user@test.com&password=12345678

Why use parameter to send email and password in POST method? I think it must on body

Maybe your backend have a problem.
The authentication are successfully, but fail when try to parse data to UserResource like the screenshot i send before

**Comment by meydiansyah** [comment]
‘cause this url with parameter just only use to mobile, so i think there is no problem with that
